package.json

```diff
@@ -39,14 +39,13 @@
     "automated-release": "release-it -VV --config ./release/.release-it.json",
     "build": "npm run build-stylesheets && rimraf ./dist/swagger-ui.js ./dist/swagger-ui.js.map && npm run build-all-bundles",
     "build-all-bundles": "run-p --aggregate-output build:core build:bundle build:standalone build:es:bundle build:es:bundle:core",
-    "build-stylesheets": "cross-env NODE_ENV=production BABEL_ENV=production BROWSERSLIST_ENV=browser-production webpack --color --config webpack/stylesheets.js",
+    "build-stylesheets": "cross-env NODE_ENV=production BABEL_ENV=production BROWSERSLIST_ENV=browser-production webpack --color --config webpack/stylesheets.js && shx sed -i \"s/\\*zoom/zoom/g\" ./dist/swagger-ui.css",
```

Comment:
Escaping double quotes due to this comment in https://github.com/shelljs/shx?tab=readme-ov-file#packagejson:

> Windows treats single quotes (ex. 'some string') differently than double quotes. https://github.com/shelljs/shx/issues/165#issuecomment-563127983 wrapping your arguments in escaped double quotes so that your code is compatible cross platform (ex. "clean": "shx echo \"some string\"").